lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/passport/issuecredential/domain/verifiablecredential/Evidence.java

```diff
@@ -1,27 +1,31 @@
-package uk.gov.di.ipv.cri.passport.library.domain.verifiablecredential;
+package uk.gov.di.ipv.cri.passport.issuecredential.domain.verifiablecredential;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonInclude;
-import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
-import uk.gov.di.ipv.cri.passport.library.annotations.ExcludeFromGeneratedCoverageReport;
+import uk.gov.di.ipv.cri.common.library.annotations.ExcludeFromGeneratedCoverageReport;
+import uk.gov.di.ipv.cri.passport.issuecredential.domain.checkdetails.Check;
 
 import java.util.List;
 
-import static uk.gov.di.ipv.cri.passport.library.domain.verifiablecredential.VerifiableCredentialConstants.EVIDENCE_TYPE_IDENTITY_CHECK;
-
-@DynamoDbBean
 @ExcludeFromGeneratedCoverageReport
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class Evidence {
 
-    private String type = EVIDENCE_TYPE_IDENTITY_CHECK;
+    private String type = "IdentityCheck";
```

Comment:
We have this as an enum should we not be using that?

Reply:
The change to using the common VerifiableCredentialClaimsSetBuilder meant the majority of VC constants could be removed.

The VerifiableCredentialClaimsSetBuilder internally uses the value VC_CREDENTIAL_TYPE_ICC = "IdentityCheckCredential"; so could not be used.

That leaves the only place IdentityCheck was needed as a string being internally in the evidence object. - Used in both VC evidence and Audit evidence.

```
        "evidence": [
            {
                "type": "IdentityCheck",
                "txn": "txn_id",
                "strengthScore": 4,
                "validityScore": 2
            }
        ]
```